Assets/Scripts/Game/Serialization/SerializablePlayer.cs

```diff
@@ -123,23 +123,14 @@ public object GetSaveData()
             data.playerEntity.globalVars = entity.GlobalVars.SerializeGlobalVars();
 
             // Store player position data
-            data.playerPosition = new PlayerPositionData_v1();
-            data.playerPosition.position = transform.position;
-            data.playerPosition.yaw = playerMouseLook.Yaw;
-            data.playerPosition.pitch = playerMouseLook.Pitch;
-            data.playerPosition.isCrouching = playerMotor.IsCrouching;
-            data.playerPosition.worldPosX = StreamingWorld.LocalPlayerGPS.WorldX;
-            data.playerPosition.worldPosZ = StreamingWorld.LocalPlayerGPS.WorldZ;
-            data.playerPosition.insideDungeon = playerEnterExit.IsPlayerInsideDungeon;
-            data.playerPosition.insideBuilding = playerEnterExit.IsPlayerInsideBuilding;
-            data.playerPosition.terrainSamplerName = DaggerfallUnity.Instance.TerrainSampler.ToString();
-            data.playerPosition.terrainSamplerVersion = DaggerfallUnity.Instance.TerrainSampler.Version;
-            data.playerPosition.weather = GameManager.Instance.WeatherManager.PlayerWeather.WeatherType;
+            data.playerPosition = GetPlayerPositionData();
```

Comment:
This was clever. Well done. :)

Reply:
Ha, thank you. It was actually lazy... as you know the best programmers are lazy ones!!